File: 백준/Silver/1758. 알바생 강호/알바생 강호.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        Integer[] tip = new Integer[n];
        for(int i = 0;i<n;i++){
            tip[i] = Integer.parseInt(br.readLine());
        }
        Arrays.sort(tip, Collections.reverseOrder());
        long ans = 0;
        for(int i = 0;i<n;i++){
            int temp = tip[i]-i;
            if (temp>0){
                ans+=temp;
            }
        
        }
        System.out.println(ans);
    }
    
}

File: 백준/Silver/1758. 알바생 강호/알바생 강호.py
import sys
input = sys.stdin.readline


n = int(input())
tip = [int(input())for _ in range(n)]
tip.sort(reverse=True)

ans = 0
for i in range(n):
    temp = tip[i]-i
    if temp>0:
        ans += temp
print(ans)